pkg/kubelet/kubelet.go

```diff
@@ -40,7 +40,7 @@ import (
 	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/api/validation"
 	"k8s.io/kubernetes/pkg/apis/componentconfig"
-	kubeExternal "k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1"
+	kubeexternal "k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1"
```

Comment:
componentconfigv1alpha1 isn't the end of the world - it's accurate, and probably easier to read than the shortened form due to lack of vowels.